st2common/st2common/services/executions.py

```diff
@@ -42,6 +44,22 @@
 
 LOG = logging.getLogger(__name__)
 
+SKIPPED = ['id', 'context', 'callback', 'action']
+
+
+def _decompose_liveaction(liveaction_db):
+    """
+    Splits the liveaction into an ActionExecution compatible dict.
+    """
+    decomposed = {'liveaction': {}}
+    liveaction_api = vars(LiveActionAPI.from_model(liveaction_db))
+    for k in liveaction_api.keys():
+        if k in SKIPPED:
+            decomposed['liveaction'][k] = liveaction_api[k]
+        else:
+            decomposed[k] = getattr(liveaction_db, k)
```

Comment:
I see your point. Luckily not much of LiveAction is stored any longer but the same would apply to other fields like Action, Rule, Trigger etc in an ActionExecution.

What do you suggest we do?